.gitignore

```diff
@@ -44,6 +44,6 @@ datasets/dev_org/test_data.db
 .cci
 
 # LWC
-/src/lwc/jsconfig.json
-/src/lwc/.eslintrc.json
+force-app/main/default/lwc/jsconfig.json
```

Comment:
I think we're including this jsconfig.json in the repo now, so it could be removed from here.